.github/workflows/build.yaml

```diff
@@ -33,17 +33,59 @@ jobs:
       - name: Install dependencies
         run: pip install -r requirements.txt
 
-      # We have two cores so we can speed up the testing with xdist
-      - name: Install xdist, openfiles and flake8 for pytest
-        run: pip install "flake8<5" pytest-xdist pytest-openfiles pytest-flake8 pytest-cov
+      - name: Install pytest packages
+        shell: bash -l {0}
```

Comment:
I don't think you need the `shell` line any more here or below.

Reply:
I think that's right.  I don't see a need for it.

Reply:
I will remove the shell lines and then make sure the checks work properly when rerun without them.